pydip/player/unit.py

```diff
@@ -23,6 +26,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+    def __lt__(self, other):
```

Comment:
From what I can tell you seemed to toss in this `__lt__` implementation in order to make the `sorted` call from `example.py` work? On the surface I'm not sure I would consider `Unit`s to be comparable like this, so perhaps it would be better to use `sorted(player.units, key=lambda unit: unit.position)`?

Reply:
Oh I think I slightly misread the `example.py` code where you were using this before -- actually adding a `Player.find_unit(territory)` does seem like a very good solution here, since that will have the added benefit of documenting exactly which unit you're selecting in the `example.py` script. 🙂